nvflare/job_config/script_runner.py

```diff
@@ -146,6 +151,7 @@ def add_to_fed_job(self, job, ctx, **kwargs):
             job.add_executor(executor, tasks=tasks, ctx=ctx)
 
         job.add_resources(resources=[self._script], ctx=ctx)
+        return comp_ids
```

Comment:
Already did. This is okay because the caller right does not use the returned result.
This is to conform to the convention that the add_to_fed_job() method should return the IDs of added components.